features/providers/civil_application_journey.feature

```diff
@@ -404,14 +438,8 @@ Feature: Civil application journeys
     Then I choose option "Respondent bail conditions set True"
     Then I fill "Bail conditions set details" with "Foo bar"
     Then I click "Continue"
-    Then I should be on a page showing "Has your client received legal help for the matter?"
-    Then I choose "No"
-    Then I fill "Application purpose" with "because I can't afford it"
-    Then I click "Continue"
-    Then I should be on a page showing "Are the proceedings, for which funding is being sought, currently, before the court?"
-    Then I choose "Yes"
-    Then I fill "Details of proceedings before the court" with "The case is already in the court"
-    Then I click "Continue"
+    And I should not see "Client received legal help"
+    And I should not see "Proceedings currently before court"
```

Comment:
Do we really need two checks here that we are not on a particular page - surely one would be fine.